system/doc/reference_manual/expressions.xml

```diff
@@ -836,6 +836,21 @@ number &lt; atom &lt; reference &lt; fun &lt; port &lt; pid &lt; tuple &lt; map
       depending on the size of the float because otherwise comparison of large
       floats and integers would lose their transitivity.</p>
 
+    <p>The term equivalence operators, <c>=:=</c> and <c>=/=</c>, return
+      whether two terms are indistinguishable. While the other operators
+      consider the same <em>numbers</em> equal regardless of whether they are
```

Comment:
I think that "regardless of" can be omitted. At least, that sounds good to me.

Reply:
Thanks, I've tweaked the wording a bit. :)